src/server/rpc/procedures/sdss/sdss.js

```diff
@@ -0,0 +1,79 @@
+// This is the SDSS RPC. It wraps the web API of the Sloan Digital Sky Survey.
+
+'use strict';
+
+var debug = require('debug'),
+    request = require('request'),    
+    CacheManager = require('cache-manager'),   
+    cache = CacheManager.caching({store: 'memory', max: 1000, ttl: Infinity}),
+    info = debug('netsblox:rpc:movie-db:info'),
+    trace = debug('netsblox:rpc:movie-db:trace');
+
+
+// Retrieving static images
+var baseUrl = 'http://skyserver.sdss.org/dr13/SkyserverWS';
+
+module.exports = {
+
+    // This is very important => Otherwise it will try to instantiate this
+    isStateless: true,
+
+    /**
+     * Return the path to the given RPC
+     *
+     * @return {String}
+     */
+    getPath: function() {
+        return '/SDSS';
```

Comment:
we should probably change this to something more user-friendly for the end user (maybe "sky-server") since SDSS is unlikely to mean anything... Thoughts?